SimCalorimetry/HGCalAssociatorProducers/interface/AssociatorTools.h

```diff
@@ -1,3 +1,9 @@
+#if !defined(SimCalorimetry_HGCalAssociatorProducers_interface_AssociatorTools_h)
+#define SimCalorimetry_HGCalAssociatorProducers_interface_AssociatorTools_h
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "SimDataFormats/CaloAnalysis/interface/CaloParticle.h"
+#include <vector>
+
 void removeCPFromPU(const std::vector<CaloParticle>& caloParticles, std::vector<size_t>& cPIndices) {
```

Comment:
Not really on the scope of this PR, but shouldn't this function be declared as `inline` (or move the implementation to source, which would probably be better)?